### ISSUE TITLE
Jars in mavenrepo broken

### ISSUE THREAD

**Issue by lake4790k** [MrPowers/spark-fast-tests#24]
Seems all jars in mavenrepo only contain html javadocs not the code, eg
https://repo1.maven.org/maven2/com/github/mrpowers/spark-fast-tests_2.11/0.12.5/
all jars have same file size

**Comment by rdefreitas** [comment]
I'm also seeing this same issue

**Comment by MrPowers** [comment]
@lake4790k @rdefreitas - Thanks for bug report.  I tried really hard to figure this out ([see here](https://stackoverflow.com/questions/50893846/maven-jar-file-only-includes-html-files-not-scala-classes)), but wasn't able to figure out why the Maven Jar files only contain HTML / JS / image files and don't contain any of the actual project code - aaaahhhh!

I will try again.  Do either of you have experience uploading JAR files to Maven with SBT.  I need help 🙏

**Comment by MrPowers** [comment]
@lake4790k @rdefreitas - I figured out how to get spark-fast-tests in Maven.  Can you try this?

```
resolvers += "Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven"
libraryDependencies += "MrPowers" % "spark-fast-tests" % "2.3.0_0.13.1"
```

Let me know if this works!

**Comment by hche608** [comment]
> @lake4790k @rdefreitas - I figured out how to get spark-fast-tests in Maven. Can you try this?
> 
> ```
> resolvers += "Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven"
> libraryDependencies += "MrPowers" % "spark-fast-tests" % "2.3.0_0.13.1"
> ```
> Let me know if this works!

Still same issue

**Comment by MrPowers** [comment]
Thanks @hche608.  

The JAR files are [here](https://mvnrepository.com/artifact/MrPowers/spark-fast-tests?repo=spark-packages).

I am going to close this issue now.  Please reopen this if you need additional help accessing JAR files.  Thanks!

**Comment by nightscape** [comment]
I know this is an old issue, but I was wondering why your artifacts have a non-standard naming scheme (the Scala version at the end).
I posted an [answer to your StackOverflow question](https://stackoverflow.com/questions/50893846/maven-jar-file-only-includes-html-files-not-scala-classes/61910635#61910635).
If this fixes the problem, it would be nice to use the standard naming scheme to make the required SBT `libraryDependencies` entry a little less confusing 😉

**Comment by MrPowers** [comment]
@nightscape - thanks for the message.  I will try to get this fixed with your suggestion.  Do you think I should follow this guide? https://leonard.io/blog/2017/01/an-in-depth-guide-to-deploying-to-maven-central/

I am open to suggestions!  Feel free to send me a pull request if you're feeling ambitious.  I really need to get this fixed for all the spark-fast-tests users!

**Comment by nightscape** [comment]
Are you currently releasing manually and would like to do automated releases?
I've set this up in https://github.com/crealytics/spark-excel using https://github.com/olafurpg/sbt-ci-release along with the following Github Workflow https://github.com/crealytics/spark-excel/blob/master/.github/workflows/ci.yml

**Comment by MrPowers** [comment]
This has been fixed thankfully, [see here for the Scala 2.12 JAR files](https://repo1.maven.org/maven2/com/github/mrpowers/spark-fast-tests_2.12/).  Thanks everyone for the help!
